### ISSUE TITLE
_AnimatedButtonState#641e1(tickers: tracking 1 ticker) was disposed with an active Ticker.

### ISSUE THREAD

**Issue by lucasw2025** [NikhilVadoliya/FlutterAnimatedButton#13]
It seems that the Animation was not disposed:
```
_AnimatedButtonState created a Ticker via its TickerProviderStateMixin, but at the time dispose() was called on the mixin, that Ticker was still active. All Tickers must be disposed before calling super.dispose().

Tickers used by AnimationControllers should be disposed by calling dispose() on the AnimationController itself. Otherwise, the ticker will leak.

The offending ticker was: _WidgetTicker(created by _AnimatedButtonState#641e1)
The stack trace when the _WidgetTicker was actually created was:
```

**To Reproduce**
1. Use the FlutterAnimatedButton to navigate to another page. I used GoRouter's context.goNamed

**Expected behavior**
It doesn't affect UI but it prints the Exception and memory may leak.

**Comment by Ahror97** [comment]
Same issue

**Comment by geanmartinezsh** [comment]
Same issue, any fix here?

**Comment by JohnettJobben1** [comment]
Hi @NikhilVadoliya, Are there any plans to update this library?

Hi @geanmartinezsh, We encountered the same issue and later discovered that you had created a fork and added a fix. Thanks for that!

For anyone else facing the same issue, you can try the following fork:

Repository: https://github.com/SportsHi/FlutterAnimatedButton.git

To use it in your project, update your `pubspec.yaml` like this:

```
flutter_animated_button:
    git:
      url: https://github.com/SportsHi/FlutterAnimatedButton.git
      ref: main
```

This should resolve the issue. Thanks again to @geanmartinezsh for the fix!

**Comment by NikhilVadoliya** [comment]
Hello @JohnettJobben1 , We have fixed this issue in the latest version `2.0.4` so i am closing this issue now.

**Comment by JohnettJobben1** [comment]
Hi, @NikhilVadoliya Thanks so much for the quick fix and the new release! Really appreciate your help.👏
